fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -71,18 +72,12 @@ public class TaskRun implements Comparable<TaskRun> {
 
     private TaskRunProcessor processor;
 
-    @SerializedName("status")
     private TaskRunStatus status;
 
-    @SerializedName("type")
     private Constants.TaskType type;
 
-    @SerializedName("executeOption")
```

Comment:
Is is compatible after upgrade?

Reply:
No. Those string only used in `inspect_task_runs` and logging.

I found it's confused if displaying too much.